client/blocks-library/Sidebar.js

```diff
@@ -0,0 +1,133 @@
+import BlocksContext from './BlocksContext';
+const { __ } = wp.i18n;
+const { TabPanel, ToggleControl, Button } = wp.components;
+
+const defaultTabs = [
+	'favorites',
+	'all-blocks',
+];
+
+
+const analogBlockSlides = wp.hooks.applyFilters( 'analogBlocks.carousel', [
+	{
+		'title': __( 'Upgrade to Style Kits Pro', 'ang' ),
+		'content': __( 'Enjoy unlimited access to the template and block library, along with many more features in Style Kits Pro.', 'ang' ),
+		'button': {
+			'link': 'https://analogwp.com/style-kits-pro/?utm_medium=plugin&utm_source=library&utm_campaign=style+kits+pro',
+			'text': __( 'Learn More', 'ang' )
+		}
```

Comment:
@lushkant Let's add another key here for `isActive` which will determine if a said item should be shown/hidden.